api/v1beta2/job_types.go

```diff
@@ -190,6 +190,10 @@ type BucketConfig struct {
 	// +kubebuilder:default=s3
 	// +optional
 	BackendType string `json:"backendType,omitempty"`
+
+	// Path to SSL CA certificate file instead of system default.
+	// +optional
+	CaCerts string `json:"caCerts,omitempty"`
```

Comment:
It seems that the certificate file is used in addtion to the system default, not instead of.

The singular form "cert" would be appropriate.

```suggestion
	// Path to SSL CA certificate file used in addition to system default.
	// +optional
	CaCert string `json:"caCert,omitempty"`
```

Reply:
fixed at https://github.com/cybozu-go/moco/pull/580/commits/b7f3575f875a74b7ca1b0ffb2aef09dec317352c